packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -247,28 +147,6 @@ export const detachExperienceStyles = (experience: Experience): string | undefin
           );
         },
       });
-      /**
-     * propsByBreakpoint {
```

Comment:
When I was reviewing the original PR, I found these comments quite helpful. Could we not remove them?
If we don't want them to bloat the code here, they could also be moved to the docstring of the respective functions that produce each data structure.
So this `propsByBreakpoint` example can be moved to the `indexByBreakpoint` function.
And the `stylesForBreakpoint` example later on can go into the doc string of builtCfStyles this way they are easy to see every time you use each function.

Reply:
I was having a hard time to visually parse the whole function. So moving it to the docstring of each function would be great. Will do that!